sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -992,6 +992,9 @@ case class FormatNumber(x: Expression, d: Expression)
 
       val sb = classOf[StringBuffer].getName
       val df = classOf[DecimalFormat].getName
+      val dfs = classOf[DecimalFormatSymbols].getName
+      val l = classOf[Locale].getName
+      val US = "US"
```

Comment:
(I am assuming you're referring to the `US` variable).
It would work to just have it inside the string below. 
I decided to define it separately purely on grounds of taste - this way, the String does not contain any names of classes or objects. Putting the `US` in would be an exception, which (IMO) would make the code slighly harder to read.